documentation/man/features/build.rst

```diff
@@ -16,6 +16,7 @@ SYNOPSIS
 | *kw* (*b* | *build*) [\--llvm] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-c | \--clean)] [\--alert=(s | v | (sv | vs) | n)]
 | *kw* (*b* | *build*) [(-f | \--full-cleanup)] [\--alert=(s | v | (sv | vs) | n)]
+| *kw* (*b* | *build*) [\--cflags] [\--alert=(s | v | (sv | vs) | n)]
```

Comment:
The `--alert` doesn't exist as of now, so it's not needed here. Maybe in a future PR, remove all mentions to this option that is spread around the code.

Reply:
Well noted, I'll change that soon. thx :))